.circleci/config.yml

```diff
@@ -21,26 +21,28 @@ jobs:
         type: string
     working_directory: ~/jest
```

Comment:
Creating a thread to discuss ☝🏼  https://github.com/facebook/jest/pull/12546#pullrequestreview-900374553